packages/lib-utils/src/app/redux/ReduxExtensionProvider.test.tsx

```diff
@@ -0,0 +1,118 @@
+import type {
+  ReduxProvider,
+  ResolvedExtension,
+  LoadedExtension,
+} from '@openshift/dynamic-plugin-sdk';
+import { useResolvedExtensions } from '@openshift/dynamic-plugin-sdk';
+import { render } from '@testing-library/react';
+import * as React from 'react';
+import type { ReactReduxContextValue } from 'react-redux';
+import { createSelectorHook } from 'react-redux';
+import type { Store } from 'redux';
+import { createStore } from 'redux';
+import ReduxExtensionProvider from './ReduxExtensionProvider';
+
+type LRReduxReducer = LoadedExtension<ResolvedExtension<ReduxProvider>>;
+
+jest.mock('@openshift/dynamic-plugin-sdk', () => ({
+  useResolvedExtensions: jest.fn(),
+}));
+
+const useResolvedExtensionsMock = jest.mocked(useResolvedExtensions, false);
+
+describe('ReduxExtensionProvider', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
I'd suggest to do any mock clear or reset in `beforeEach` instead of `afterEach`.

Note that we already have
```js
clearMocks: true,
```
in `packages/common/test-configs/jest-config-base.ts` file.

I personally prefer to be specific about how and when a test suite clears or resets its mocks, like it's done above, instead of a global `clearMocks: true` setting (which I'd like to remove in the future).